frontend/pages/media/publications.js

```diff
@@ -71,14 +76,21 @@ const Publications = () => {
                       <h1>{year}</h1>
                       <div className={styles.row}>
                         {articles
-                          .filter((article) => article.year === year)
-                          .map((article, index) => (
+
+                          .filter(
+                            (publication) =>
+                              publication.fields.year === year.toString()
+                          )
+
+                          .map((publication, index) => (
                             <PubArticle
                               key={index}
-                              imgUrl={`/publications/${year}/${article.img}`}
-                              heading={article.heading}
-                              date={article.date}
-                              url={article.url}
+                              imgUrl={
```

Comment:
Images are also not being rendered in preview. Refactor logic to account for the missing article covers 

![image](https://user-images.githubusercontent.com/80564580/189946233-001dbae3-a018-4725-b268-125432233915.png)

Reply:
I'm not sure why, but when I open it up in netlify preview, all the images are there, and when I tested it locally was well. And all the images are on contentful

<img width="1438" alt="Screen Shot 2022-09-16 at 9 46 33 am" src="https://user-images.githubusercontent.com/44431750/190527630-a223db7a-1809-45b5-bf9c-b7b15c394c38.png">